src/main/java/com/twilio/http/HttpUtility.java

```diff
@@ -0,0 +1,32 @@
+package com.twilio.http;
+
+import com.twilio.Twilio;
+import lombok.experimental.UtilityClass;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@UtilityClass
+class HttpUtility {
+    String getUserAgentString(final List<String>... userAgentExtensions) {
+        StringBuilder userAgentString = new StringBuilder();
+        userAgentString.append("twilio-java/")
+                .append(Twilio.VERSION)
+                .append(" (")
+                .append(Twilio.OS_NAME)
+                .append(Twilio.OS_ARCH)
```

Comment:
Should there be a separator between these 2?

Reply:
addressed. Added a space.